doc/man1/openssl.pod

```diff
@@ -589,7 +589,7 @@ B<s_client> and B<s_server> commands.
 
 =item B<-dtls>, B<-dtls1>, B<-dtls1_2>
 
-These options specify to use DTLS instead of DLTS.
+These options specify to use DTLS instead of TLS.
```

Comment:
Reviewers: This part of the fix I am not 100% certain about.

Reply:
The fix is correct.